project/plugins.sbt

```diff
@@ -30,7 +35,7 @@ addSbtPlugin("uk.gov.hmrc" % "sbt-artifactory" % "1.0.0")
 
 addSbtPlugin("uk.gov.hmrc" % "sbt-distributables" % "2.0.0")
 
-addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.5.19")
+addSbtPlugin("com.typesafe.play" % "sbt-plugin" % "2.6.23")
```

Comment:
This can be bumped back up to the latest version (2.6.25) after our conversation seeing as we don't have to try and get around the mongo dependency issue.